landlab/bmi/bmi_bridge.py

```diff
@@ -260,7 +263,7 @@ def initialize(self, fname):
             clock_params = params.pop("clock")
             self._clock = TimeStepper(**clock_params)
 
```

Comment:
@mcflugen I'm confused. Isn't ~L261 the place where we need to use the new `create_grid`?

Reply:
Right. Good point. I spent all that time refactoring `create_grid` but don't actually use it. 🤦‍♂ .